easybuild/easyconfigs/e/exiv2/exiv2-0.27.5-GCCcore-11.3.0.eb

```
@@ -0,0 +1,34 @@
+easyblock = 'CMakeMake'
+
+name = 'exiv2'
+version = '0.27.5'
+
+homepage = 'https://exiv2.org'
+description = """
+ Exiv2 is a C++ library and a command line utility to manage image metadata. It provides fast and easy read and write
+ access to the Exif, IPTC and XMP metadata of digital images in various formats. Exiv2 is available as free software and
+ with a commercial license, and is used in many projects.
+"""
+
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ['https://github.com/Exiv2/exiv2/releases/download/v%(version)s/']
+sources = ['%(namelower)s-%(version)s-Source.tar.gz']
+checksums = ['35a58618ab236a901ca4928b0ad8b31007ebdc0386d904409d825024e45ea6e2']
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.23.1'),
+]
+
+dependencies = [
+    ('expat', '2.4.8'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/exiv2', 'lib/libexiv2.%s' % SHLIB_EXT, 'lib/libexiv2-xmp.a', 'lib/pkgconfig/exiv2.pc'],
+    'dirs': ['include/exiv2', 'lib/cmake'],
+}
+
```

Please keep sanity check commands of previous easyconfig
```suggestion

sanity_check_commands = ["exiv2 --help"]

```